tests/unit/datepicker/methods.js

```diff
@@ -21,6 +21,15 @@ QUnit.test( "destroy", function( assert ) {
 	assert.equal( dp.css( "display" ), "block", "Datepicker - visible" );
 	inp.datepicker( "hide" ).datepicker( "destroy" );
 	assert.ok( $.datepicker._curInst == null, "Datepicker - destroyed and cleared reference" );
+	assert.equal( dp.css( "display" ), "none", "Datepicker - absent" );
+
+	// Destroy Without Hiding
```

Comment:
Can we use something more describing`? Maybe like:

`// Destroy without manual hiding (ensure datepicker is hidden after calling destroy)`

Reply:
Let's do it now:
```suggestion
	// Destroy without manual hiding (ensure datepicker is hidden after calling destroy)
```
(yes, I'll squash it when merging)

Reply:
Perfect!